### PR TITLE
✨ Migrate chrome extension to manifest v3

### PR DESCRIPTION
**Pull request by powerivq**
Upgrade all essential features to manifest v3.
- Change background page to a service worker
- Use locally hosted wasm AMP validator
- Various manifest item upgrade

### CODE REVIEW

File: validator/js/chromeextension/worker.js [hunk 1]
```diff
@@ -281,113 +282,73 @@ function validateUrlFromTab(tab, userAgent) {
...
-  const url = tab.url.split('#')[0];
-  xhr.open('GET', url, true);
-
-  // We can't set the User-Agent header directly, but we can set this header
-  // and let the onBeforeSendHeaders handler rename it for us.
-  // Add the listener now. It'll be removed after the request is made.
-  // It's not possible to set filters on the listener to only capture our
-  // traffic, so this approach will interfere as little as possible with the
-  // 99.9% of requests which aren't for AMP validation.
-  chrome.webRequest.onBeforeSendHeaders.addListener(
```
[the remark this answers is not in the document]
Comment (powerivq) [reply validator/js/chromeextension/worker.js:294]: We switched from using xhr to fetch API. Fetch API natively supports setting headers and therefore we do not need to rely on this workaround.

**Review by banaag — APPROVED** [review]

**Comment by powerivq** [comment]
@erwinmombay wanna get your feedback before closing since this change is not trivial.

**Review by erwinmombay — APPROVED** [review]

File: validator/js/chromeextension/build_extension.sh [hunk 2]
```diff
@@ -2,6 +2,8 @@
 #
 # A script to install dependencies and then build the extension.
 
+curl -f https://cdn.ampproject.org/v0/validator_wasm.js -o validator_wasm.js
```
Comment (erwinmombay) [inline validator/js/chromeextension/build_extension.sh:5]: when does `build_extension.sh` get called btw?

Comment (powerivq) [reply validator/js/chromeextension/build_extension.sh:5]: Everything is manual as of now. Need to call this every time to manually build the extension.

**Pull request merged**